tests_integration/__tests__/plugin-resolution.js

```diff
@@ -1,22 +1,108 @@
 "use strict";
```

Comment:
It seems the current implementation does not support `pluginSearchDirs` in config files, which we should do something like the following in `resolve-config.js`:

```js
options.pluginSearchDirs = options.pluginSearchDirs.map(
  pluginSearchDir => path.resolve(
    path.dirname(configFilePath), 
    pluginSearchDir
  )
)
```